test/hotspot/gtest/aarch64/aarch64-asmtest.py

```diff
@@ -16,8 +16,8 @@
      = [0x1, 0x38, 0x7e, 0xff, 0x1fc, 0x1ff, 0x3f0,
         0x7e0, 0xfc0, 0x1f80, 0x3ff0, 0x7e00, 0x7e00,
         0x8000, 0x81ff, 0xc1ff, 0xc003, 0xc7ff, 0xdfff,
-        0xe03f, 0xe10f, 0xe1ff, 0xf801, 0xfc00, 0xfc07,
-        0xff03, 0xfffe]
+        0xe03f, 0xe1ff, 0xf801, 0xfc00, 0xfc07, 0xff03,
```

Comment:
Removed "0xe10f" from the list as it is not a valid 16-imemdiate. 
Source : https://gist.github.com/dinfuehr/51a01ac58c0b23e4de9aac313ed6a06a